content/integrations/AWS-Integration/AWS-CloudFormation-Installation.md

```diff
@@ -12,31 +12,24 @@ This installation method leverages a CloudFormation script which can be viewed [
 
 After you make a new AWS integration in CloudWisdom, the script populates a read-only IAM role in your AWS account and links it using the integration’s Account ID and External ID. Once created, it may take a few minutes for the status to be updated.
 
-## Create an AWS Integration in CloudWisdom
-1. Open CloudWisdom and navigate to Integrations > AWS.  
-2. Create a new AWS integration. Ensure that the below tabled properties are set according to your needs.
+## 1. Create an AWS Integration in CloudWisdom
+1. Open CloudWisdom and navigate to **Integrations** > **Amazon Web Services**.  
+   - If this is not your first AWS integration, select **+ Add Integration**.
+   ![add-integration](/images/AWS-CloudFormation-Installation/add-integration.png)
+3. Enable the **Cost Explorer API**.
```

Comment:
We should add a note that only one datasource can have this toggle enabled as only one Cost Explorer API connection per tenant is supported. They would get an error if another datasource has it already enabled but we can save them the trouble of finding out then with a note here.

Reply:
good point! I added a note and clarified this should really be done specifically for a master billing accnt